tardis/utilities/utils.py

```diff
@@ -98,6 +99,15 @@ def machine_meta_data_translation(
         raise
 
 
+def str_to_state(resources):
```

Comment:
I'm somewhat stumped by this function. Looking at the name, I would have assumed it to take a *single* state *name* and look that up. Perhaps a name such as `load_states` would work better? Do you see benefit in a single-name-lookup `str2state` in `dronestates`, or is that too trivial?

Reply:
Yes, `load_states`sounds much better. I have renamed it accordingly. I think an additional `str2state` function in `dronestates` is not reasonable, because the database plugin always returns a listing. In that case with a single element. So, I could either use `load_states` and get the first entry in the list or use a `str2state` function and put in the first entry from the database listing.  So, it is "gehupft wie gesprungen". ;-)